### PR TITLE
Pyu/afp 302 add breadcrumps to help with navigation

### PR DESCRIPTION
**Pull request by pyupew**

### CODE REVIEW

File: packages/website/src/components/breadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  padding: ${math.multiply(gridSize, 4)}px ${math.multiply(gridSize, 5)}px;
+  padding-bottom: ${math.multiply(gridSize, 1.5)}px;
+`;
+
+type BreadcrumbsNavProps = {
+  pagePath: string;
+};
+
+const BreadcrumbsNav = ({ pagePath }: BreadcrumbsNavProps) => {
+  const pgPath = pagePath.replace('/index.js', '').replace('.js', '');
```
Comment (Noviny) [inline packages/website/src/components/breadcrumbs.tsx]: Where do we have pagePaths that include `.js` - that seems like a bug elsewhere in the system.

Comment (Noviny) [reply packages/website/src/components/breadcrumbs.tsx]: Similar, are all the paths absolute, or do we get relative paths? I assume this only works if we get absolute paths.

Comment (pyupew) [reply packages/website/src/components/breadcrumbs.tsx]: this is the absolute file path. The file extension gets replaced in `getImportPath` when converting to a relative path, but I'm grabbing `pagePath` before that happens so I am doing the replacement in the breadcrumbs component

Comment (Noviny) [reply packages/website/src/components/breadcrumbs.tsx]: Hmkay. Does this imply we should lift that conversion higher up?

**Comment by pyupew** [comment]
> Hm. We already had some weird jumping in top positioning between initial load and the second paint - this seems to be worse now, as the first breadcrumbs item doesn't jump, but the others do. I have no idea why.
> 
> I'm treating this as part of that existing bug, not a blocking part of this PR, but does amp the priority of that task.

There's some weird discrepancy between CSR and SSR styling, so the jump is being caused by initially seeing the SSR styles before they are overridden by the CSR styles. The breadcrumbs are doing a server-side transition which is why we're seeing them jump. I'm trying to replace the components with Next Links, but this is removing all styles. Going to try and fix it, but if not then will just merge as is.

File: packages/website/src/components/breadcrumbs.tsx (hunk 1 above)
Comment (pyupew) [reply packages/website/src/components/breadcrumbs.tsx]: Possibly, but I'm not sure if that'll break anything

**Pull request merged**